Exceptions/CloudApiQuotaExceeded.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Piwik\Plugins\GoogleAnalyticsImporter\Exceptions;
+
+class CloudApiQuotaExceeded extends \RuntimeException
+{
+    public function __construct()
+    {
+        parent::__construct('Daily cloud rate limit reached, try again tomorrow. (Note: GA by default sets a daily limit on the number'
+            . ' of API requests made each day to 50000. This limit is prorated for each Matomo cloud customer.
```

Comment:
@elabuwa The no "50000" looks misleading, since its way less what is allowed based on account.
We can directly mention the allowed request for trial and paid customer here

Reply:
Thank you @AltamashShaikh . I've updated the error message a bit as well. @tsteur, is the error message ok?

Reply:
Will users see this message?

And yes ideally we show the actual number of allowed requests. FYI we don't need to mention "cloud" in the exception. Technically, the plugin doesn't even know it's overwritten by cloud. We could just say "The daily quota of X requests has been reached. We will try to continue again tomorrow." maybe.